include/tvm/build_module.h

```diff
@@ -126,10 +127,52 @@ struct BuildConfig {
   /*! \brief Whether to partition const loop */
   bool partition_const_loop = false;
 
+  void VisitAttrs(AttrVisitor* v) final {
+    v->Visit("data_alignment", &data_alignment);
+    v->Visit("offset_factor", &offset_factor);
+    v->Visit("double_buffer_split_loop", &double_buffer_split_loop);
+    v->Visit("auto_unroll_max_step", &auto_unroll_max_step);
+    v->Visit("auto_unroll_max_depth", &auto_unroll_max_depth);
+    v->Visit("auto_unroll_max_extent", &auto_unroll_max_extent);
+    v->Visit("unroll_explicit", &unroll_explicit);
+    v->Visit("restricted_func", &restricted_func);
+    v->Visit("detect_global_barrier", &detect_global_barrier);
+    v->Visit("partition_const_loop", &partition_const_loop);
+  }
+
+  static constexpr const char* _type_key = "BuildConfig";
+  TVM_DECLARE_NODE_TYPE_INFO(BuildConfigNode, Node);
+};
+
+/*! \brief BuildConfig, contains options for a build. */
+class BuildConfig : public NodeRef {
```

Comment:
There is a shortcut, see https://github.com/dmlc/tvm/blob/master/include/tvm/target_info.h#L43